default_module_facts.yml

```diff
@@ -1,2 +1,4 @@
 ---
 pe_version: '2016.4.1'
+puppet_environmentpath: '/etc/puppetlabs/code/environments'
+puppet_vardir: '/opt/puppetlabs/puppet/cache'
```

Comment:
I honestly don't know, I added to an existing file, I assumed it's in the right place
let me move it